pkg/controller/podgroup.go

```diff
@@ -266,3 +276,17 @@ func (ctrl *PodGroupController) patchPodGroup(old, new *schedv1alpha1.PodGroup)
 	}
 	return nil
 }
+
+func fillOccupiedObj(pg *schedv1alpha1.PodGroup, pod *v1.Pod) {
+	refs := make([]string, 0)
+	for _, ownerRef := range pod.OwnerReferences {
+		refs = append(refs, fmt.Sprintf(pod.Namespace, ownerRef.Name))
+	}
+	if len(pg.Status.OccupiedBy) == 0 {
+		return
+	}
+	if len(refs) != 0 {
+		sortkeys.Strings(refs)
```

Comment:
Is it equivalent to `sort.Strings(refs)`? If yes, let's eliminate the dependency on `sortkeys`.

Reply:
removed

Reply:
I meant you can still sort them, but we can just use the default `sort` library.

Reply:
done